newsfragments/3262.misc.txt

```diff
@@ -0,0 +1,3 @@
+Since the L2 network is always implied based on the TACo network connected to, we no longer need those config/parameters across the codebase, it can be inferred.
```

Comment:
just curious why this is a .txt file instead of .rst

Reply:
Whoops!  Thanks for noticing.  Renamed it ✔️

Reply:
Nice catch @cygnusv , thanks for fixing @KPrasch .